app/backend/src/couchers/servicers/conversations.py

```diff
@@ -530,6 +534,10 @@ def CreateGroupChat(self, request, context, session):
             ).scalar_one_or_none():
                 context.abort(grpc.StatusCode.FAILED_PRECONDITION, errors.ALREADY_HAVE_DM)
 
+        # Check if user has been initiating chats excessively
+        if has_reached_chat_initiation_limit(session=session, user_id=context.user_id):
```

Comment:
It's not clear now that this is not a pure function based on the name. I don't know what to call it. A really bad idea is `process_rate_limit_return_should_abort`

Reply:
Yeah, this is a good point. Was thinking to break it up, but I think from the flow it makes sense like this, so just renamed it.